### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by thongdong7** [DevExpress/testcafe#1696]
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

```bash
$ testcafe chrome tests/t1.ts
ERROR Cannot prepare tests due to an error.

TypeError: Cannot read property 'length' of undefined

Type "testcafe -h" for help.
```

### What is the expected behavior?

The testcase could be execute without error.

### How would you reproduce the current behavior (if this is a bug)?

```bash
$ yarn global add testcafe
$ testcafe --version
0.17.0
$ testcafe chrome tests/t1.ts
```

* When downgrade to 0.16.x, it works fine.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: http://devexpress.github.io/testcafe/example

Test code

```js
import { Selector } from 'testcafe';

fixture `Getting Started`.page `http://devexpress.github.io/testcafe/example`;

test('My first test', async t => {
  // Test code
});
```

### Specify your

* operating system: MacOS
* testcafe version: 0.17.0
* node.js version: 8.3.0_1

**Comment by AlexanderMoskovkin** [comment]
I @thongdong7,

Thanks for the report. I've reproduced the issue on MacOS. We'll fix it in the next minor version.

**Comment by AlexanderMoskovkin** [comment]
closed in #1698
